modules/network-observability-updating-migrating.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/installing-operators.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-updating-migrating_{context}"]
+= Migrating removed stored versions of the CRD
+
+Network Observability Operator version 1.6 removes the old and deprecated `v1alpha1` version of the `FlowCollector` API. If you previously installed this version on your cluster, it might still be referenced in the `storedVersion` of the `FlowCollector` CRD, even if it is removed from the etcd store, which blocks the upgrade process. These references need to be manually removed.
+
+Several options exist to do this, and the following steps ensure that the installation is in a clean state. You can alternatively use the Storage Version Migrator Operator. For more information, see _Kubernetes Storage Version Migrator Operator_.
+
+.Prerequisites
+* you have an older version of the Operator installed, and you want to prepare your cluster to install the latest version of the Operator. Or you have attempted to install the Network Observability Operator 1.6 and run into the error: `Failed
+risk of data loss updating "flowcollectors.flows.netobserv.io": new CRD removes version v1alpha1 that is listed as a stored version on the existing CRD`.
+
+.Procedure
+. Verify that the old `FlowCollector` CRD version is still referenced in the `storedVersion`:
++
+[source,terminal]
+----
+$ oc get crd flowcollectors.flows.netobserv.io -ojsonpath='{.status.storedVersions}'
+----
+. If `v1alpha1` appears in the list of results, proceed with Step 3-6 to fully uninstall and reinstall the CRD and the Operator.
+. Save the Network Observability Operator 1.5 version of the `FlowCollector` CR to a file, for example `flowcollector-1.5.yaml`. 
++
+[source,terminal]
+----
+$ oc get flowcollector cluster -o yaml > flowcollector-1.5.yaml
+----
+. Follow the steps in "Uninstalling the Network Observability Operator", which uninstalls the Operator and removes the existing `FlowCollector` CRD.
```

Comment:
Is it better to link to the Uninstalling section here?

Reply:
With the docs, we have this weird limitation with links where we can't link inline. The best I can do is an `Additional resource` heading, but I didn't do that here because the Uninstalling procedure is on this page already. So I thought I'd just refer to it like this since you can see it in the left nav. 🤷‍♀️

Reply:
yup saw other sections and its the same way as this, so no need to add a link. Looks good!